keyboards/chlx/piche60/readme.md

```diff
@@ -0,0 +1,30 @@
+# piche60
+
+![piche60 PCB](https://i.imgur.com/CBeJSUy.jpeg)
+
+- 60% alice format keyboard pcb with USB-C and unified daughterboard compatibility.
+- Layout adds 3 additional keys in the palm area voids of traditional alice format.
+- USB-C can be snapped off and converted into an ai03 C3 compatible daughterboard.
+- Left and right halves can be snapped off and connected through 14-pin JST-SH interfaces.
+- PCB can be manufactured with top-side USB-C for compatibility with legacy alice PCBs. 
+
+Keyboard Maintainer: [Alexander Lee](https://github.com/gaclee3b)
+Hardware Supported: chlx PCB piche60.x.x.x
```

Comment:
No link to a store page, or repo for pcb fabrication?

Reply:
I usually just run orders through r/mk bulk sale posts and google forms, but haven't run one yet for this. do I just link that later or are you asking for an official store of sorts before the merge is approved?

Reply:
A mention of that should be fine, I think.